rskj-core/src/main/java/org/ethereum/core/genesis/GenesisLoaderImpl.java

```diff
@@ -260,4 +252,24 @@ public static void loadGenesisInitalState(Repository repository, Genesis genesis
             repository.updateAccountState(accountEntry.getKey(), accountEntry.getValue());
         }
     }
+
+    private static GenesisJson readGenesisJson(InputStream inputStream) {
```

Comment:
Maybe this method doesn't need to be static since it's only called from the constructor

Reply:
well, this method does not deal with an object's state, that's why I marked it as static one